### ISSUE TITLE
Sphinx monospace font looks too similar to normal font in html

### ISSUE THREAD

**Issue by jongeorge1999** [spacepy/spacepy#774]
Input parameters are put into mono-space when referencing input parameters in function/class doc-strings, however, there is not a large enough discernible difference between the fonts to notify users that some words are keywords.

![image](https://github.com/user-attachments/assets/a251cc7f-e4f7-4f77-bbb8-858203ad7680)
Example where "data" and "type" are both input parameters and should be distinguishable from non-keywords

Spacepy 0.7.0a

**Comment by jtniehof** [comment]
Deferred until after #759
